crates/bdk/src/wallet/tx_builder.rs

```diff
@@ -27,7 +31,7 @@
 //!     // Create a transaction with one output to `to_address` of 50_000 satoshi
 //!     .add_recipient(to_address.script_pubkey(), 50_000)
 //!     // With a custom fee rate of 5.0 satoshi/vbyte
-//!     .fee_rate(FeeRate::from_sat_per_vb(5.0))
+//!     .fee_rate(bdk::FeeRate::from_sat_per_vb(5.0))
```

Comment:
I didn't look super closely but you should be able to get rid of your `FeeRate` now and use `bitcoin::FeeRate`.

Reply:
I opened an issue for this one: https://github.com/bitcoindevkit/bdk/issues/1037 as I feel it needs more discussion: at the moment `bdk::FeeRate` has a couple methods that `bitcoin::FeeRate` doesn't have, and although switching is possible, I think we should discuss whether it's better to switch immediately or to first submit a patch to you guys :)